<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gsDockWidget</class>
 <widget class="QDockWidget" name="gsDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChatGIS</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLineEdit" name="messageEdit"/>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="sendBtn">
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QPlainTextEdit" name="chatEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="clearBtn">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>messageEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>sendBtn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>577</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>582</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
